File: 4.study/12.function/1.new/6.Login_Ruby/1-1.rb
# members = ['gst', 'kst']
# for member in members do
#     if member == input_id
#         puts('Hello, '+member)
#         exit
#     end
# end
# puts('Who are you?')

# 로그인 애플리케이션(루비)
puts("입력해주세요.\n")
input_id = gets.chomp()

def login(id)
    members = ['gst', 'kst']
    for member in members do
        if member == id
            return true
        end
    end
    return false
end

if login(input_id)
    puts('Hello, '+input_id)
else
    puts('Whoi are you?')
end

File: 4.study/12.function/1.new/7.Login_python/1-1.py
input_id = input("아이디를 입력해주세요.\n")
def login(_id):
    members = ['gst', 'kst']
    for member in members:
        if member == _id:
            return True
    return False

if login(input_id):
    print('Hello, '+input_id)
else:
    print('Who are you?')
